src/responder/whatsapp.py

```diff
@@ -784,8 +784,13 @@ def send_correction_poll_expert(self, row_lt, expert_row_lt, row_query, escalati
         
         receiver = expert_row_lt["whatsapp_id"]
         forward_to = expert
+        try:
+            gender = row_lt['patient_gender'][0].upper()
+            patient_details = f"\n*Patient*: {row_lt['patient_name']} {row_lt['patient_age']}/{gender}\n"
+        except:
+            patient_details= ""
 
-        poll_text = f'*Query*: "{row_query["message_english"]}" \n*Bot\'s Response*: {row_bot_conv["message_english"].strip()} \n\n*User*: {user_type} \n*Citations*: {final_citations.strip()}. \n\n{poll_string}'
+        poll_text = f'*Query*: "{row_query["message_english"]}" \n*Bot\'s Response*: {row_bot_conv["message_english"].strip()} \n{patient_details}*Citations*: {final_citations.strip()}. \n\n{poll_string}'
```

Comment:
[Minor]
There will be extra line when patient_details is emply
Can new line be avoided in this case ?

Reply:
I already took that into account, new line is within the try statement